apps/language_server/lib/language_server/providers/definition.ex

```diff
@@ -9,9 +9,11 @@ defmodule ElixirLS.LanguageServer.Providers.Definition do
 
   def definition(uri, text, line, character) do
     case ElixirSense.definition(text, line + 1, character + 1) do
+      # FIXME: Private module
```

Comment:
This is a bug in ElixirSense - Location should be a public module

Reply:
Okay, makes sense. I have removed the annotation on this side. When we make it public I think we should shorten the name, maybe change it from `ElixirSense.Providers.Definition.Location` to just `ElixirSense.Location`.